code/modules/surgery/bodyparts/parts.dm

```diff
@@ -67,7 +67,7 @@
 	is_dimorphic = FALSE
 	wound_resistance = -10
 	bodytype = BODYTYPE_MONKEY | BODYTYPE_ORGANIC
-	acceptable_bodytype = BODYTYPE_HUMANOID
+	acceptable_bodytype = BODYTYPE_MONKEY
```

Comment:
This was changed in https://github.com/tgstation/tgstation/pull/73325 and it seems to have been intentional, Monkeys were given support to use human limbs.

Perhaps it would be better to merge monkey bodytype into humanoid?